tracing-mock/Cargo.toml

```diff
@@ -17,9 +17,14 @@ edition = "2018"
 rust-version = "1.49.0"
 publish = false
 
+[features]
+default = []
+subscriber = ["tracing-subscriber"]
```

Comment:
hmm, maybe we should just have the feature be the optional `tracing-subscriber` dep, instead of adding a separate feature flag? we could put `#[cfg(feature = "tracing-subscriber")]` on the code that requires the flag. i could be convinced either way, depends on what you think would be clearer to users...

Reply:
I don't have a strong opinion here either, so I'm going with the simpler approach and leaving the `[features]` section out completely and using `#[cfg(feature = "tracing-subscriber")]` as suggested.